packages/codemods/src/helpers/tests/either.test.ts

```diff
@@ -0,0 +1,109 @@
+import { Left, Right, Either, DoEither } from '../either';
+
+const getEither = <T, Z>(right: T, left: Z, returnLeft: boolean = false): Either<T, Z> => {
```

Comment:
let's split this to `goLeft` and `goRight` , that is going to make the test a lot easier to read